src/main/java/uk/gov/hmcts/ccd/endpoint/exceptions/CasePersistenceException.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.ccd.endpoint.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.CONFLICT)
```

Comment:
perhaps we can find a better error code than 409 for a PersistenceException?

Reply:
412 could be another option, but I think 409 is the best in the case of version conflict in opt locking.